### ISSUE TITLE
Issue with environment object - Fatal error: No ObservableObject of type TestViewModel found.

### ISSUE THREAD

**Issue by SumoSimo** [nalexn/ViewInspector#77]
Hi nalexn, I'm running into an issue with a test involving an environment object. You can see the test below and associated modules. I followed approach 2, and I'm not sure what the issue could be.

```
          let mockTestViewModel = MockTestViewModel()
          stub(mockTestViewModel) { stubTestViewModel in
            when(stubTestViewModel.boolean.set(any())).thenCallRealImplementation()
          }
          
          let subject = TestView()
          
          let expectation = subject.inspection.inspect { view in
            let button = try view.find(button: "Tap me")
            try button.tap()
            
            let foo = try? view.find(TestView2.self)
            expect(foo != nil).to(beTrue())
          }
          
          ViewHosting.host(view: subject.environmentObject(mockTestViewModel))
          self.wait(for: [expectation], timeout: 1)
```

```
struct TestView: View {
  
  @EnvironmentObject private var testViewModel: TestViewModel
  
  let inspection = Inspection<Self>()
  
  var body: some View {
    if testViewModel.boolean {
      TestView2()
    } else {
      ZStack {
        VStack {
          Text("Whoa, I'm high up")
          
          Spacer()
        }
        
        VStack {
          Spacer()
          
          Text("I'm down low")
        }
        
        Button("Tap me") {
          testViewModel.boolean.toggle()
        }
      }
      .onReceive(inspection.notice) { self.inspection.visit(self, $0) }
    }
  }
  
}

struct TestView2: View {
  
  var body: some View {
    Text("I am test view 2")
  }
  
}
```

```
class TestViewModel: ObservableObject {
  
  @Published var boolean = false
  
}

class MockTestViewModel: TestViewModel, Cuckoo.ClassMock { ... }
```

I've also tried using `TestViewModel` instead of `MockTestViewModel` in the test to no avail.

**Comment by nalexn** [comment]
Sorry for the long response, this just got fixed as of v0.7.0

**Comment by SumoSimo** [comment]
@nalexn Good to hear. Thanks!
